### PR TITLE
JS API/Editor/Panel: Fix - Double render on 'panel/editor/open' (Fix …

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: assets/dev/js/editor/views/controls-stack.js [hunk 1]
```diff
@@ -51,9 +51,16 @@ ControlsStack = Marionette.CompositeView.extend( {
 		},
 	},
 
-	initialize: function() {
+	initialize: function( options ) {
 		this.initCollection();
 
+		// Handles issue with globalSelect - pt 850.
```
Comment (matipojo) [inline assets/dev/js/editor/views/controls-stack.js:57]: Remove the comment.

File: assets/dev/js/editor/views/controls-stack.js [hunk 2]
```diff
@@ -51,9 +51,16 @@ ControlsStack = Marionette.CompositeView.extend( {
...
+		if ( options.tab ) {
+			this.activeTab = options.tab;
+			// Depends on initCollection.
```
Comment (matipojo) [inline assets/dev/js/editor/views/controls-stack.js:60]: What does it mean?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/views/controls-stack.js (hunk 2 above)
Comment (iNewLegend) [reply assets/dev/js/editor/views/controls-stack.js]: Handled.

File: assets/dev/js/editor/views/controls-stack.js (hunk 1 above)
Comment (iNewLegend) [reply assets/dev/js/editor/views/controls-stack.js]: Handled.

**Review by matipojo — DISMISSED** [review]

**Review by kobizz — APPROVED** [review]

**Pull request merged**